.deployment/group_vars/all/vars.yml

```diff
@@ -0,0 +1,13 @@
+opencast_admin_password: '{{ lookup("env", "OPENCAST_ADMIN_PASSWORD") }}'
+opencast_digest_password: '{{ lookup("env", "OPENCAST_DIGEST_PASSWORD") }}'
+opencast_letsencrypt_email: '{{ lookup("env", "OPENCAST_LETSENCRYPT_EMAIL") }}'
+opencast_postgresql_password: '{{ lookup("env", "OPENCAST_POSTGRESQL_PASSWORD") }}'
+
+# repository configuration
+opencast_version_major: 10
+opencast_repository_enabled_release: true
+opencast_repository_enabled_testing: true
+
+# path configuration
+opencast_data_path: /mnt/share
+opencast_storage_downloads_path: '{{ opencast_data_path }}/opencast/downloads'
```

Comment:
Lots of these variables aren't used anywhere... in the scripts you added here. Are your ansible galaxy scripts just access these variables? I would have assumed one has to somehow explicitly pass them. Not sure if I'm a fan of these global variables with magic names. But that's not a complaint about this PR :D

Reply:
Nono it's fine, that seems like an established Ansible way to do things. So no docs in this PR required.